wallet-new/src/Cardano/Wallet/API/V1/Migration/Types.hs

```diff
@@ -168,6 +169,10 @@ instance Migrate (V0.CId V0.Addr, V0.CCoin) V1.PaymentDistribution where
         pdAmount  <- eitherMigrate cCoin
         pure $ V1.PaymentDistribution {..}
 
+instance Migrate V1.PaymentDistribution (V0.CId V0.Addr, Core.Coin) where
```

Comment:
Ditto as above.

Reply:
There is no counter part `Migrate (V0.CId V0.Addr, Core.Coin) V1.PaymentDistribution`. Note `Core.Coin` vs. `V0.CCoin`

Reply:
same as here https://github.com/input-output-hk/cardano-sl/pull/2107#discussion_r156997007

Reply:
Just for the record https://github.com/input-output-hk/cardano-sl/pull/2107#discussion_r157008005